new_basics/app/rectangles.js

```diff
@@ -0,0 +1,21 @@
+function area(rectangle) {
+  return rectangle.width * rectangle.height
+};
+
+function perimeter(rectangle) {
+  return rectangle.width + rectangle.height
```

Comment:
Is this the entire perimeter? Shouldn't it be rectangle.width * 2 + rectangle.height * 2